spec/support/fill_in_the_form_steps.rb

```diff
@@ -106,6 +116,18 @@ def and_has_submited_their_address
     end
   end
 
+  def and_has_submited_the_miminum_address_information
+    expect(page).to have_content(I18n.t("coronavirus_form.questions.support_address.title"))
+    within find(".govuk-main-wrapper") do
+      fill_in "building_and_street_line_1", with: "Test Please Ignore"
+      fill_in "building_and_street_line_2", with: ""
+      fill_in "town_city", with: "Test Please Ignore"
+      fill_in "county", with: ""
```

Comment:
Can this line be removed from the test? It feels odd "filling in" something that the user hasn't touched.

Reply:
I believe the empty string 'fill in' on line 123 has to be there to make the test in any way meaningful. I'm not sure exactly how it works but I suspect the validation doesn't perform any checks if you omit the 'fill_in' for a field. For example line 122, which is definitely required, If i remove that line the tests still pass whereas if I give it an empty string it fails. 

I believe this is true for the other lines with empty strings that you highlighted.